### PR TITLE
Youtube Clone Project

### PR DESCRIPTION
**Pull request by ElijahTobs**
The completed version of youtube clone project with my partner @tGodson

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+        </button>
+    </div>
+
+    <button class="topright"><img id="upload" class="header-image" src="./img/upload-vid.jpg" alt="upload video"></button>
+
+    <button class="topright"><img id="ytapps" class="header-image" src="./img/ytapp.png" alt="youtube apps"></button>
+
+    <button class="topright"><img id="options" class="header-image" src="./img/ellipsis.png" alt="options"></button>
+      
+    <button id="sign-in_btn">Sign in</button>
```
Comment (adaorachi) [inline index.html]: Here, you have an overuse of the `id` attribute. The purpose of which you have them can be interchanged with the `class `attribute. The id attribute is used only when it is needed and its use is very specific. If you want to add a unique identifier to an element, you can do that by adding multiple classes, give one of them a unique name and use that class name for styling. Kindly, remove the `id` attribute and implement the suggestion above as they are not needed.

Comment (adaorachi) [inline index.html]: Here, you have an overuse of the `id` attribute. The purpose of which you have them can be interchanged with the `class `attribute. The id attribute is used only when it is needed and its use is very specific. If you want to add a unique identifier to an element, you can do that by adding multiple classes, give one of them a unique name and use that class name for styling. Kindly, remove the `id` attribute and implement the suggestion above as they are not needed.

**Comment by adaorachi** [comment]
👋 Hi @ElijahTobs and @tGodson.

Well-done submitting your first Microverse project!🎇. I really want to commend the hard work you have put into it and the features you implemented. The nice use of different thumbnail images and the properly-aligned main and aside sections. In all, your project looks aesthetically nice. 💯 

But there are few changes to be made in order for your project to look very professional and just like the original site. 
So hang in there, you're just a step away from completing this project! 💪 

---
### **CHANGES REQUIRED** ❗️ 
---

- [x] Your HTML codes have a number of errors. Make sure to check your codes using this validator - [validator.w3](https://validator.w3.org/#validate_by_upload) and fix those errors.
---
- [x] From the image below, you have all elements in the navbar in their different containers and which **should not** be. Your navbar should have three containers. The first one should contain the toggle-bar and logo and aligned to the leftmost part of the navbar. The second should contain the search bar and button and aligned to the center. And the third should have the icons and sign-in button and aligned to the rightmost part. Kindly fix it.

**Image:**

Annotation 2020-03-26 234534

---
- [x] Your README file should contain a short description of your project and a screenshot of your project too. Do update the content of what you have in the image below with the right contents.
**Image:**

Annotation 2020-03-26 23453411
---

File: index.html [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+            <img src="./img/search-icon.jpg" alt="search icon" id="search-icon">
+        </button>
+    </div>
+
+    <button class="topright"><img id="upload" class="header-image" src="./img/upload-vid.jpg" alt="upload video"></button>
+
+    <button class="topright"><img id="ytapps" class="header-image" src="./img/ytapp.png" alt="youtube apps"></button>
+
+    <button class="topright"><img id="options" class="header-image" src="./img/ellipsis.png" alt="options"></button>
+
```
Comment (adaorachi) [inline index.html]: Here, you have an overuse of the `id` attribute. The purpose of which you have them can be interchanged with the `class `attribute. The id attribute is used only when it is needed and its use is very specific. If you want to add a unique identifier to an element, you can do that by adding multiple classes, give one of them a unique name and use that class name for styling. Kindly, remove the `id` attribute and implement the suggestion above as they are not needed.

**Review by adaorachi — CHANGES_REQUESTED** [review]

**Comment by adaorachi** [comment]
---
**After implementing the changes above, request for another review by submitting a code review form.**

---

The following requirement is **not mandated** for this project but it'll nice to implement as this would make your project look more professional. 

- [x] [**optional**] You should consider making the contents in the aside section that contains the thumbnail images and title clickable. You can achieve this by enclosing each content in an anchor tag `<a>`.

- [x] [**optional**] You should always make use of git's best practice such as committing codes very often. It helps you keep your code updated with the latest changes so that you avoid conflicts. It is easier to track changes and roll back a particular feature you do not want in your code. It also helps you from losing work, reverting changes, and helping trace what you did.

Should you have any questions or need any help, feel free to reach out to me on Slack(@adaorachi).

**Good-luck in it and Happy Coding** 💪 

**MaryAnn Chukwuka**

**Comment by ElijahTobs** [comment]
My partner @tGodson and I have made the requested changes.

**Comment by lekegitrepo** [comment]
### Hi @ElijahTobs and @tGodson 👏 💯 🥇 
### You did an excellent job of implementing the required changes. This Project is approved. Good luck with the next one.

### Note: Please remember to add live demo link in the readme.

### Reviewed by: _Temitayo Adeleke_

-   Github: [@lekegitrepo](https://github.com/lekegitrepo)
-   Twitter: [@leketemi](https://twitter.com/leketemi)
-   Linkedin: [linkedin](https://www.linkedin.com/in/temitayo-adeleke/)

**Review by lekegitrepo — APPROVED** [review]
### Status: Approved

**Pull request merged**